pkg/transform/kubernetes/kubernetes.go

```diff
@@ -38,6 +38,13 @@ func createServices(app *spec.App) []runtime.Object {
 		if len(svc.Spec.Selector) == 0 {
 			svc.Spec.Selector = app.Labels
 		}
+		if s.Name == "" {
```

Comment:
@kadel I'd want that too, but at one place I am checking `len(app.PersistentVolumes)` and at another it's `len(app.Services)`, how do you suggest I abstract that?